f8a_worker/dispatcher/flows/stackApiGraphFlow.yml

```diff
@@ -1,14 +1,19 @@
 ---
   flow-definitions:
-    - name: 'stackApiGraphFlow'
+    - name: 'stackApiGraphV2Flow'
```

Comment:
Why ? From the rest of the change it looks like we're removing `stackApiGraphV2Flow`

Reply:
Good catch!